ports/gstreamer/portfile.cmake

```diff
@@ -252,42 +166,25 @@ if (NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
     endif()
 endif()
 
-vcpkg_fixup_pkgconfig()
-
-# Remove duplicated GL headers (we already have `opengl-registry`)
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/include/KHR"
-                    "${CURRENT_PACKAGES_DIR}/include/GL"
-)
-file(RENAME "${CURRENT_PACKAGES_DIR}/lib/gstreamer-1.0/include/gst/gl/gstglconfig.h"
-            "${CURRENT_PACKAGES_DIR}/include/gstreamer-1.0/gst/gl/gstglconfig.h"
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/lib/${pkg_name}/pkgconfig"
+                    "${CURRENT_PACKAGES_DIR}/lib/${pkg_name}/pkgconfig"
 )
+vcpkg_fixup_pkgconfig()
 
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share"
-                    "${CURRENT_PACKAGES_DIR}/debug/libexec"
-                    "${CURRENT_PACKAGES_DIR}/debug/lib/gstreamer-1.0/include"
-                    "${CURRENT_PACKAGES_DIR}/libexec"
-                    "${CURRENT_PACKAGES_DIR}/lib/gstreamer-1.0/include"
-)
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/bin"
                         "${CURRENT_PACKAGES_DIR}/bin"
     )
-    set(PREFIX ${CMAKE_SHARED_LIBRARY_PREFIX})
-    set(SUFFIX ${CMAKE_SHARED_LIBRARY_SUFFIX})
-    file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/${PREFIX}gstreamer-full-1.0${SUFFIX}"
-                "${CURRENT_PACKAGES_DIR}/lib/${PREFIX}gstreamer-full-1.0${SUFFIX}"
-    )
-endif()
-if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
-    file(GLOB DBG_BINS "${CURRENT_PACKAGES_DIR}/debug/lib/gstreamer-1.0/*.dll"
-                       "${CURRENT_PACKAGES_DIR}/debug/lib/gstreamer-1.0/*.pdb"
+elseif(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
+    file(GLOB DBG_BINS "${CURRENT_PACKAGES_DIR}/debug/lib/${pkg_name}/*.dll"
```

Comment:
I think we should make a patch to fix that.

Reply:
I will check if it can be configured in meson.build

Reply:
I couldn't find a way to separate `bin` and `lib` with the project's build option. :(

I studied a little. [`library` has an option `install_dir`](https://mesonbuild.com/Reference-manual_functions.html#arguments36), and it becomes the destination of the library files, DLLs, and PDB.

Reply:
I will take a look tomorrow.

Reply:
No I don't know why gstreamer does this but probably versioned installed dirs are a good default if you don't install versioned libs ?